app/Models/EventAdmin.php

```diff
@@ -34,11 +34,11 @@ class EventAdmin extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->